noxfile.py

```diff
@@ -147,6 +147,34 @@ def test_integration_cloud(session):
     )
 
 
+@nox.session(
+    venv_backend=VENV_BACKEND,
+    python=environ.get("PYTHON_VERSION", "3.11"),
+)
+def test_integration_sqlachemy_v1(session):
```

Comment:
I was expecting the code that ran redsfhit/snowflake to be deleted. how are you preventing snowflake and redshift tests from being executed?

Reply:
I have removed `integration-test-live` from the `.github/workflows/ci.yaml` file which ensures the snowflake / redshift tests are not invoked. I didnt delete the workflow or function because as u mentioned we might need to test later if user faces any issues.

Reply:
Added a comment for future reference: https://github.com/ploomber/jupysql/pull/982/commits/ef7e3777eefe92d8dda47bc1d12d52341bb60857